en/guide/follow_me.md

```diff
@@ -52,10 +51,12 @@ if (follow_me_result != FollowMe::Result::SUCCESS) {
     std::cout << "Failed to start following:" << FollowMe::result_str(follow_me_result) << std::endl;
 }
 
+
 // ... Get target position from underlying platform or wherever and supply to vehicle 
 // Here we just show one point being set. Minimum values specified in the configuration are respected.
 follow_me.set_curr_target_location({ 47.39776569, 8.54553292, 9.0, 0.f, 0.f, 0.f });

```

Comment:
@hamishwillee altitude here is not of significance. This is altitude of the target. We've already specified Vehicle's altitude in FollowMe configuration.

Reply:
Yes, thanks.